### ISSUE TITLE
retina compatible icons

### ISSUE THREAD

**Issue by fuddl** [openstreetmap/openstreetmap-website#850]
on the way to retina support (#104), one thing that has to be done is adding high res icons.

I mean these:
![osm icons](https://cloud.githubusercontent.com/assets/842548/5280619/ed069450-7af3-11e4-8694-64a7f76d24a7.png)

are these available as svg somewhere? or as a font?

**Comment by Zverik** [comment]
Is this issue still relevant?

**Comment by bcherny** [comment]
yes - icons still look bad on retina:

<img width="62" alt="screen shot 2015-09-10 at 11 11 01 am" src="https://cloud.githubusercontent.com/assets/1761758/9796799/d1050610-57ac-11e5-82ce-e355c239d758.png">

can we just serve the SVG instead of PNG? svg is:

- [supported by all modern browsers](http://caniuse.com/#feat=svg)
- future compatible with higher dpi devices that may appear in the near future
- smaller (just 1.1kb gzipped after running through svgo) compared to PNG (3.3kb gzipped)

**Comment by HolgerJeromin** [comment]
I am pretty sure osm.org should be usable with Internet Explorer < 9. 
Thats why i [suggested](https://github.com/openstreetmap/openstreetmap-website/issues/850#issuecomment-65446147) a compatible version.

**Comment by bcherny** [comment]
can you make a pr?

**Comment by HolgerJeromin** [comment]
Made two PR for the two changes and thrown both SVGs into SVGO (nice tool btw :-)
The directions button has no SVG file in the repo. Perhaps @zerebubuth can help here.

The marker (for routing and other stuff) were not changed. These are interacting with the map via drag and drop and i was not brave enough for changing them, too.

**Comment by HolgerJeromin** [comment]
could we close this or should we wait for 
https://github.com/openstreetmap/openstreetmap-website/blob/master/app/assets/images/directions.png

**Comment by tomhughes** [comment]
Well this is unfortunately the kind of ticket that is not sufficiently concrete to be easily closeable. There are many image that are still not retina compatible, and even some of the ones we have added patches for are not really sorted.

In particular I discovered after I merged your `picture` based fix for the logo that one of the two main browsers where retina support is needed (iOS Safari) doesn't actually support `picture` elements at all, so we really need to revisit that.

**Comment by HolgerJeromin** [comment]
You are right. Safari seems to ignore picture element, but rely on the uglier srcset for img tag. I hoped this is a temporary problem, but the [bug report](https://bugs.webkit.org/show_bug.cgi?id=116963) is quite old. 
IEs support for picture is [under development](https://dev.modern.ie/platform/status/pictureelement/), but srcset on img is [ready now](https://dev.modern.ie/platform/status/imgsrcset/).

**Comment by danieldegroot2** [comment]
@gravitystorm Requesting to label these issues at least as `ui` to make it easier to find.

**Comment by hiddewie** [comment]
I made a pull request in https://github.com/openstreetmap/openstreetmap-website/pull/4775 to replace a number of commonly used icons. If this pull request is appreciated, I can replace many more of the icons with their SVG variants. This should improve the retina support of the OpenStreetMap website considerably.

**Comment by AntonKhorev** [comment]
Some markers (routing, user locations) are still pngs. They should be similar to note markers from #4797.

**Comment by hlfan** [comment]
Do you think the marker on the export page needs to be an SVG as well?
And if so, should it be a separate file or defined in the export view?

**Comment by hlfan** [comment]
> Some markers (routing, user locations) are still pngs. They should be similar to note markers from #4797.

Is there anything left after #5854?
